src/map/skill.cpp

```diff
@@ -10079,6 +10079,11 @@ int32 skill_castend_nodamage_id (struct block_list *src, struct block_list *bl,
 
 	case TF_BACKSLIDING: //This is the correct implementation as per packet logging information. [Skotlex]
 		{
+			// Backsliding makes you immune to being stopped for 200ms, but only if you weren't immune already
+			unit_data* ud = unit_bl2ud(bl);
+			if (ud != nullptr && DIFF_TICK(ud->endure_tick, tick) < 0)
+				ud->endure_tick = tick + 200;
```

Comment:
Not really a friend of that DIFF_TICK syntax, would prefer simply using std::max or i64max

Reply:
I think we should consistently use DIFF_TICK mainly because then we can easily do adjustments that should affect all t_tick comparisons.

Reply:
I think i dont really understand your explanation.
But if i understand you correctly, then we should introduce a utility function that does this.